ui/app/mainui/SendModalHandler.qml

```diff
@@ -148,87 +149,171 @@ QtObject {
     **/
     required property var fnGetOpenSeaUrl
 
+    /** property to store the params to be updated in the send modal until it is launched **/
+    property var simpleSendParams
+
     /** signal to request launch of buy crypto modal **/
     signal launchBuyFlowRequested(string accountAddress, int chainId, string tokenKey)
 
-    function openSend(params = {}) {
+    function openSend(params = {}, forceLaunchOldSend = false) {
```

Comment:
why do we need `forceLaunchOldSend`

Reply:
in case of bridge we still use the old send even though the feature flag for Simple send is enabled, thats why